docker-compose.dev.yml

```diff
@@ -0,0 +1,91 @@
+version: '2.3'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: ./Dockerfile.dev
+      args:
+        STATIC_URL: '/static/'
```

Comment:
How about this

```
  web:
    build:
      context: .
      dockerfile: ./Dockerfile.dev
      args:
        STATIC_URL: '/static/'
        HOST_UID="$(id -u)"
        HOST_GID="$(id -g)"
```

Note: untested but I think something like this should be ok. Will this then work without the makefile ?
It would be great if this PR worked with and without the makefile.

Reply:
Yea, I just tried the function `id -u` and it won't accept it.

An alternative is 

```
  web:
    build:
      context: .
      dockerfile: ./Dockerfile.dev
      args:
        STATIC_URL: '/static/'
        HOST_UID: ${UID}
        HOST_GID: ${GID}
```

and I think that will work. Gonna confirm it now.

Reply:
Ok by omitting default values args will take on the value of whatever variable has been defined in the environment with the same name.

So:

```
STATIC_URL: '/static/'
HOST_UID:
HOST_GID:
```

```
export HOST_UID=$(id -u) && export HOST_GID=$(id -g) && \
docker-compose --file docker-compose.dev.yml build --no-cache
```

would do the trick.